Makefile

```diff
@@ -22,17 +22,7 @@ include Makefile.common
 
 # For users who don't read the INSTALL file
 .PHONY: defaultentry
-defaultentry:
-ifeq "$(UNIX_OR_WIN32)" "unix"
-	@echo "Please refer to the installation instructions in file INSTALL."
-	@echo "If you've just unpacked the distribution, something like"
-	@echo "	./configure"
-	@echo "	make world.opt"
-	@echo "	make install"
-	@echo "should work.  But see the file INSTALL for more details."
-else
-	@echo "Please refer to the instructions in file README.win32.adoc."
-endif
```

Comment:
The error message below has been less relevant since 4.08, since if you forget to run `configure`, then these two `include`s cause an error message and this one isn't reached.

This message could be moved to a target for `Makefile.config` and `Makefile.common` advertising the need to run `configure`, i.e.

```make
Makefile.%:
	@echo "Please refer..."
	@false
```